sdks/python/apache_beam/ml/inference/tensorflow_inference_it_test.py

```diff
@@ -62,7 +64,9 @@ def test_tf_mnist_classification(self):
         save_main_session=False)
     self.assertEqual(FileSystems().exists(output_file), True)
 
-    expected_output_filepath = 'gs://clouddfe-riteshghorse/tf/mnist/output/testing_expected_outputs_test_sklearn_mnist_classification_actuals.txt'  # pylint: disable=line-too-long
+    expected_output_filepath = (
+        'gs://clouddfe-riteshghorse/tf/mnist/output/testing_expected_outputs_test_sklearn_mnist_classification_actuals.txt'  # pylint: disable=line-too-long
```

Comment:
Can we move these out of your test account and into `apache-beam-ml`? https://pantheon.corp.google.com/storage/browser/apache-beam-ml/testing?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&e=-13802955&jsmode=o&mods=dataflow_dev&project=apache-beam-testing&prefix=&forceOnObjectsSortingFiltering=false

Reply:
Done